### PR TITLE
Allow arbitrary git repo mode and group

### PR DESCRIPTION
**Pull request by jyaworski**

### CODE REVIEW

File: manifests/params.pp [hunk 1]
```diff
@@ -198,6 +198,12 @@
   # Dynamic environments config, ignore if the git_repo is 'false'
   # Path to the repository
   $server_git_repo_path       = "${vardir}/puppet.git"
+  # mode of the repository
+  $server_git_repo_mode       = '0755'
+  # user of the repository
+  $server_git_repo_user       = $user
+  # group of the repository
+  $server_git_repo_group       = $user
```
Comment (ekohl) [inline manifests/params.pp]: Nitpick: odd alignment here

**Comment by ekohl** [comment]
Untested, but the code looks :+1: other than the nitpick

File: manifests/params.pp (hunk 1 above)
Comment (jyaworski) [inline manifests/params.pp]: Fixed.

**Comment by jyaworski** [comment]
Fixed.

**Comment by ekohl** [comment]
:+1:

**Comment by mmoll** [comment]
merged, dzięki @jyaworski!

**Comment by jyaworski** [comment]
@mmoll dziękuje. :)

**Pull request merged**